### ISSUE TITLE
Cannot install any version later than 1.9.2-23 on Windows 11 23H2

### ISSUE THREAD

**Issue by AnnoyingMoose** [abandonware/noble#334]
```
PS C:\Users\[me]\source\repos\[project]> npm i @abandonware/noble@1.9.2-24
npm ERR! code 1
npm ERR! path C:\Users\[me]\source\repos\[project]\node_modules\@abandonware\noble
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node-gyp-build
npm ERR! 'C:\Program' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

npm ERR! A complete log of this run can be found in: C:\Users\[me]\AppData\Local\npm-cache\_logs\2024-05-27T14_05_09_287Z-debug-0.log
PS C:\Users\[me]\source\repos\[project]> npm i @abandonware/noble@1.9.2-25
npm ERR! code 1
npm ERR! path C:\Users\[me]\source\repos\[project]\node_modules\@abandonware\noble
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node-gyp-build
npm ERR! 'C:\Program' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

npm ERR! A complete log of this run can be found in: C:\Users\[me]\AppData\Local\npm-cache\_logs\2024-05-27T14_05_55_501Z-debug-0.log
```
However, 1.9.2-23 installs and works (bugs aside) just fine.

**Comment by JovannMC** [comment]
This seems to be related to `node-gyp-build` and not `@abandonware/noble`, I had this as well and was fixed by installing `node-gyp-build@4.8.0` as a dev dependency.
Refer to this: https://github.com/prebuild/node-gyp-build/issues/71

**Comment by AnnoyingMoose** [comment]
Ah, I see. Thank you for the workaround! Closing this issue, as it is not a problem with `@abandonware/noble`, as @JovannMC stated.
